packages/rum-core/src/domain/rumEventsCollection/resource/resourceCollection.ts

```diff
@@ -84,7 +87,17 @@ function processRequest(
     tracingInfo,
     correspondingTimingOverrides,
     indexingInfo,
-    responseDurationInfo
+    responseDurationInfo,
+    isExperimentalFeatureEnabled('resource_durations')
+      ? {
+          _dd: {
+            computed_duration: duration,
+            performance_entry_duration: correspondingTimingOverrides
+              ? correspondingTimingOverrides.resource.duration
+              : undefined,
+          },
+        }
+      : undefined
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what’s best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** maybe lift this into a variable before
 `const resourceEvent = combine(` 
WDYT?

Reply:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** I added a diff and diff percentage variable on another [PR](https://github.com/DataDog/browser-sdk/pull/1810), is that something that you think might be useful?

Reply:
I lifted it. 
I'd say that without your fix, I could have fallen into wrong conclusions, comparing computed and entry resource duration. So in that sense it helps me doing my investigation :)